common/prometheus-server/values.yaml

```diff
@@ -241,6 +241,8 @@ thanos:
     service:
       httpPort: 9090
       grpcPort: 10901
+    # Timeout for queries
+    #timeout: 2m
```

Comment:
I added it commented so that it was clear it was an available option. The intention is not to change the default for everyone, but allow the default to be configurable. If you prefer a different syntax for that idea, that's fine.

Reply:
I have no preferences, but helm does. That's why the test failed :)
Which values did you have in mind? Still not getting why `2m` isn't suitable?